### DISCUSSION TITLE
Add a task and have it show up on journal?

### DISCUSSION THREAD

**Discussion by jungle-boogie** [zim-desktop-wiki/zim-desktop-wiki#2430] [Q&A]
Hello,

Suppose you have a page with tasks like such:
```
[] laundry <2023-08-06
[] pay mortgage <2023-08-08
[] oil change for car <2023-10-10
```

Is there a way for those items above to appear on the journal for the respective dates?

I know I can go to each journal page and add this ahead of time, but I'd like to have a single page where those events are added to the journal. That way when i'm using the journal on that date, I'll see the entry.

thanks!

**Comment by jungle-boogie** [comment]
Is it possible for these items to appear on the journal?

**Comment by bruvellu** [comment]
As far as I'm aware, that's not possible. The closest to this functionality is having the task list as a side pane. All the tasks in your notebook will show up there and can be ordered by date:

![image](https://github.com/zim-desktop-wiki/zim-desktop-wiki/assets/361179/ba135c36-92da-4b3a-84eb-711bc2cf0d1a)

**Comment by jungle-boogie** [comment]
Thanks, @bruvellu. That I am familiar with.
